### PR TITLE
Git clone and development install of swift-zipkin

### PR DESCRIPTION
**Pull request by clayg**

### CODE REVIEW

**Comment by clayg** [comment]
As best as I was able to test this upgrades cleanly.

File: cookbooks/swift/recipes/setup.rb [hunk 1]
```diff
@@ -124,7 +124,6 @@
 if node['zipkin'] then
   [
     "git+https://github.com/swiftstack/py_zipkin/",
```
Comment (clayg) [inline cookbooks/swift/recipes/setup.rb]: I guess we could make a case this fork should also be cloned into dev

OTOH, depending on the nature of the diff we might consider if we want to outsource maintenance of this dependency - the maintainer seems to be doing releases:

https://pypi.org/project/py-zipkin/

Comment (tipabu) [reply cookbooks/swift/recipes/setup.rb]: I think things are to the point that we can use upstream now -- but it looks like we want the `work-with-py-zipkin-upstream-0.19.0` branch of swift-zipkin now? I'll poke at it a bit.

**Comment by clayg** [comment]
@tipabu I did a fresh bring up and this is working

upstream zipkin is 0.20.0
and the swift-pyzipkin patch is merged -> https://github.com/swiftstack/swift-zipkin/pull/1

**Pull request merged**